### DISCUSSION TITLE
ESP32 Stromversorgung über Pylontech

### DISCUSSION THREAD

**Discussion by DerHavey1994** [hoylabs/OpenDTU-OnBattery#1012] [Q&A] [answered]
Hallo zusammen,

ich benutze schon länger OpenDTU-onBattery mittlerweile mit 3x Victron MPPT & Pylontech. Ich würde gerne meinen ESP direkt per Step-Down Wandler an die Bus Bars bzw wenn es geht an den Pylontech anschließen, um dem unnötigen USB Stecker aus dem Weg zu gehen & alles etwas aufgeräumter zu montieren.

Als Info: Ich benutze die Platine von @LukasVFL99

Hat da jemand eine Modellbezeichnung, einen Link oder ähnliches für mich, ich möchte kein Teil kaufen, was nach 1 Woche schon den Geist aufgibt...

& dann noch eine Frage, muss ich den Step-Down Wandler an den 5V oder an den 3,3V PIN anstecken/anlöten?

Danke schon mal für eure Hilfe!

LG DerHavey

**Comment by DonJohnLong** [comment, marked as answer]
Hallo, ich habe das bei mir auch mit einem Stepdown Konverter gelöst.

Habe zuerst einen Lm2596hvs verwendet, da er aber ziemlich knapp an der max. Eingangsspannung betrieben wurde und vielleicht einmal eine kurze Überspannung sah, nun leider im Elektronik Himmel weilt. (48V Batterie) mein cmt2300 Sendemodul, hatte er auch gleich mitgenommen.

Ich verwende nun dieses Modul, was seit einem Monat absolut zuverlässig läuft:
https://de.aliexpress.com/item/1005005974953701.html
Es kann laut Verkäufer bis 120V Input vertragen, getestet habe ich bis 100V und es ist nicht explodiert 👍 

Angeschlossen wird es am 5V Input des ESP32
![20240530_185752](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/130904315/fc55779e-16bf-4072-a2d2-ac53fbeee606)

**Reply by DerHavey1994** [reply]
Vielen Dank, hab das Teil mal bestellt !

**Reply by DerHavey1994** [reply]
Habe nun das Teil eingebaut, funktioniert ohne Probleme!

In meinem Post von gestern sprach ich über komische MQTT "Mitteilungen" in Home Assistant.  #1044 Das ist jetzt durch den Wandler auch noch weg! Da hat das Netzteil (obwohl ich mehrere versucht habe..) wohl die Spannungsspitzen vom NRF nicht standgehalten.

**Comment by DonJohnLong** [comment]
Das freut mich doch, wenn es funktioniert. 👍 

Läuft bei mir auch weiterhin absolut flawless. Es ist einfach darauf zu achten, dass man die Batterie nicht bis zur abschaltgrenze des Bms entlädt.

Aber gross unter 10% sollte man anyway nicht fahren wegen der Batterie lebensdauer.

Lg

**Reply by DerHavey1994** [reply]
Tatsächlich komme ich derzeit meistens nachts nicht mal unter 20% bei der Batterie!
Die nutzbaren 4,5 kWh der Pylontech muss man auch erstmal schaffen 😂

Problematisch wird's im Winter, aber da stelle ich SOC Minimum auf 30% um genug Reserve zu haben.

**Comment by Snusme83** [comment]
Versorge die OpenDTUonBattery bzw. das Breakoutboard mit Energie von der Busbar (PV-Module, Pylontechs).

Absolut feine Sache:

Busbar--> Stepdown- Wandler (~ 48 V in auf 7V out) --> am Breakoutboard mit DC 2.1 Stecker angeschlossen. ESP32-S3 wird mit Strom versorgt. Die Pins am Breakoutboard werden dementsprechend mit 3,3 V und 5 Volt versorgt:
![IMG_2744](https://github.com/user-attachments/assets/ca3849e3-f212-4ea1-8a0d-76956cec21cc)

**Comment by DonJohnLong** [comment]
Ich bin mittlerweile zurück auf usb netzteil... Zumindest in verbindung mit einem jk bms und einfacher serieller verbindung, raucht das zeug ab.

Galvanische trennung fehlt und hat bei mir verglühte jumperkabel verursacht.

Einfach als warnung für andere user, die diese kombi einsetzen wollen, wollte ich dass noch gesagt haben. 😅

**Reply by SW-Niko** [reply]
Ich betreibe seit Monaten einen Stepdown DCDC-Wandler um die OpenDTUOnBattery mit Energie aus der Batterie zu versorgen. 
Das ist aber auch meine einzige direkte Masseverbindung zur OpenDTUOnBattery.
Dein beschriebenes Problem taucht immer auf wenn es mehrere Masseverbindungen zwischen den Geräten gibt. 
In deinem Fall "Batterie / JK-BMS <-> OpenDTUOnBattery".
Genauso verehrend ist "Batterie <-> Laderegler", "Batterie <-> OpenDTUOnBattery" und "Laderegler <-> OpenDTUOnBattery".
Das steht auch irgendwo im Wiki aber in Verbindung mit Invertern.

@Manos1966 Sollen wir das Problem nochmal deutlich ins Wiki aufnehmen?

**Reply by Snusme83** [reply]
Mehrere Masseverbindungen? Die hätte ich ja dann auch?🤔
3 x Victron SmartSolar 100/20,
1 x Breakoutboard,

Oder ist das bei mir kein Problem, weil der ESP32-S3 vom Breakoutboard mit Strom versorgt wird? Das Breakoutboard wird via DC 2.1- Stecker mit einer Spannung von 7 Volt durch den Step-Down-Wandler  von den Pylontechs versorgt!

**Reply by DonJohnLong** [reply]
Bei den ladereglern habe ich natürlich adum1201 verbaut aber für 48v konnte ich nichts gleichwertiges finden. 

Ohne galvanische trennung ist der betrieb mit stepdown wandler leider nicht save. Sie stellen gar ein sicherheitsrisiko dar.

Daher wäre dies für zukünftige user wohl hilfreich zu wissen.

**Reply by Snusme83** [reply]
Könntest du das Risiko genauer beschreiben?

**Reply by DonJohnLong** [reply]
Als ich in den ferien war, hat vermutlich mal der bms die batterie von der masse getrennt, genaue gründe dafür sind mir nicht bekannt da dann noch sommer war. 

Jedenfalls hat vermutlich dann ein wechselrichter strom verlangt und die eigentlich getrennte masse über den bms port/esp32/stepdown hergestellt. 

600w+ über jumperkabel zu jagen dürfte wohl bei jedem zu ändliche ergebissen führen wie bei mir. 😅

Zumindes geben die dünnen kabel dann guete schmelzsicherungen ab. Je nach setup besteht aber akute brandgefahr!

Solange es dafür keine lösung gibt, empfehle ich den betrieb über usb netzteil.

**Reply by spcqike** [reply]
@Snusme83 
Wenn du bspw den laderegler per ve.direct ohne adum verwendest, hast du eine direkte Masse Verbindung von der openDTU zum laderegler. 
dann die DTU per stepdown an die Batterie. Wieder eine Masse. 

Der laderegler hängt natürlich auch per Masse an der Batterie. 

Hier sind dann alle Maßen gleich. Und der Strom fliesst von laderegler über die dtu über den stepdown zur Batterie. Das ist halt fatal. 

Das umgeht man mit galvanischer Trennung. Sei es in der datenleitung (adum) oder in der Spannungsversorgung (Trafo Netzteil bzw isolierter dcdc wandler)

**Reply by DonJohnLong** [reply]
Sonnst ist das ergebniss hier zu sehen:
![20240802_085455](https://github.com/user-attachments/assets/5500bfeb-1d9c-4238-8921-d7da2941511a)

**Reply by Snusme83** [reply]
Ok, jetzt verstehe ich den Sinn der galvanischen Trennung in der Theorie. Aber ich hab ja fest verkabelt Wechselrichter - Busbar und Busbar - Pylontech.

Problematisch könnte es werden, wenn das Minus Kabel abgesteckt wird, beschädigt (quasi gekappt wird)! Dann würde sich unter Umständen quasi der Wechselrichter die Energie über den CAN- GND holen, oder?

**Comment by Snusme83** [comment]
@spcqike:

Problematisch könnte es also werden, wenn die Verkabelung (Minus) vom Laderegler zur Busbar gekappt werden würde, und aufgrund dieser fehlenden Verbindung  Minus über den Ve- Direct gezogen wird?

**Reply by SW-Niko** [reply]
Genau so ist es. 
Das geht jahrelang gut, aber wenn die starke Masse Verbindung Laderegler - Batterie ausfällt dann brennt schnell die sekundäre Masseverbindung ab. Wenn man Glück hat nur das Kabel.
Sehr oft ist das BMS schuldig. Die haben (aus Kosten Gründen) meistens "Lowside Schalter" verbaut. Also MOSFET in der Masse.
Oder man folgt bei Wartungsarbeiten dem Auto-Leitfaden. "Immer erst die Masse von der Batterie trennen".

**Reply by Snusme83** [reply]
Würde es eigentlich Sinn machen, wenn man sämtliche Massen/ Minus/ Ground die am ESP hängen mit einer Sicherung zu versehen?

Sollte es mal zu einem Sicherheitsrisiko kommen, aus welchem Grund auch immer, dann würde halt die Sicherung draufgehen, anstatt dass irgendwelche Drähte glühen!🤔

**Reply by SW-Niko** [reply]
Meine Meinung dazu ist eher Nein.
Die richtige Dimensionierung einer thermischen Schmelz Sicherung ist nicht trivial.
Meine Empfehlung bei Doppel oder Mehrfachverbindungen
- Steuerungen galvanisch trennen
- Stromversorgung über Stepdown-Wandler

Jetzt nicht falsch verstehen, es gibt sicher einfachere Lösungen, aber nicht jeder der sich eine Anlage zusammenbaut hat einen Master in Elektrotechnik. Darum empfehle ich Lösungen die möglichst immer funktionieren und die wenig Restrisiko haben.

**Comment by Manos1966** [comment]
> Würde es eigentlich Sinn machen, wenn man sämtliche Massen/ Minus/ Ground die am ESP hängen mit einer Sicherung zu versehen?

Stattdessen einfach ADUM um VE.Direkt zu schützen installieren 😉 

Wenn die Batterie den ESP32 mit Strom versorgt, die Masse wird bis zu den anderen Komponenten geleitet.
Wenn ich mich richtig erinnere, nur VE.Direkt hat eine Masse die weiter geleitet (angeschlossen) wird. Darum galvanische Trennung.
Der Rest (CANbus, NRF24, CMT2300) hat keine Masse Weiterleitung (Anschluss zu anderen Komponenten). 
Man kann (?) also davon ausgehen, dass keine Ströme weiterfliessen könnten falls die Haupt-Masse aus fehlt.
Ich hoffe, ich irre mich nicht 🙊 

EDIT: Oops, ich habe JK-BMS vergessen: "JK BMS und einfache serielle Verbindung"

**Comment by Tomi200469** [comment]
Hallo, ich benutze dafür ein Meanwell PSD-15C-5

**Reply by SW-Niko** [reply]
Ok, also galvanische Trennung auf der DC Versorgungsseite. Das funktioniert natürlich auch. 👍

**Reply by Snusme83** [reply]
Also mit einem Adum 1201 pro Laderegler und einem Adum 1201 zwischen ESP32-S3 und Step-Down- Wandler wäre man quasi auf der sicheren Seite! 🤔

**Reply by SW-Niko** [reply]
Äh.... ein ADUM1201 ist zum galvanischen trennen von digitalen Steuerleitungen. Damit kannst du keine galvanische Trennung mit einem Step-down Wandler realisieren. 
Google mal nach DC-DC Wandlern mit galvanischer Trennung. Da findest du bestimmt was passendes.

![grafik](https://github.com/user-attachments/assets/5f69791d-ba68-495b-829f-eb55cfe52327)

**Reply by Snusme83** [reply]
Da fehlt mir eben der Master in Elektrotechnik.

Für die Laderegler (VE.direct) laut Manos1966 scheinbar die adäquateste Lösung.

Für den Stepdown- Wandler funktioniert das mit dem Adum 1201 also nicht.

Gibt es irgendwo einen vernünftige Anschlussplan für Laien, damit man logisch die galvanische Trennung verstehen kann?

**Reply by SW-Niko** [reply]
>Gibt es irgendwo einen vernünftige Anschlussplan für Laien, damit man logisch die galvanische Trennung verstehen kann?

Willst du wissen wie eine galvanische Trennung funktioniert oder wann und warum sie hier benötigt wird?

**Reply by Snusme83** [reply]
Warum sie benötigt wird: 
Damit unsere Hütte nicht abfackelt, nur weil der Strom nicht über die dafür vorgesehenen Kabel fließt, sondern über nicht dafür vorgesehene viel zu dünne Adern.

Wann wird sie benötigt:
Bei mehreren Verbindungen des Mikrocontrollers mit Gerätschaften wie:
- PV- Speicher mit billigen BMS (gehört Pylontech auch dazu?)
- Ladereglern mit Ve.direct
- nicht isolierte DC- /DC- Wandler

Galvanische Trennung: Im Falle des Falles, dass große Stromstärken über nicht dafür vorgesehene Kabel fließen wird auf deutsch gesagt diese Leitung gekappt, oder?

Funktionsweise eines Adum 1201 wäre für mich interessant um zu verstehen können!

**Reply by Manos1966** [reply]
> Gibt es irgendwo einen vernünftige Anschlussplan für Laien, damit man logisch die galvanische Trennung verstehen kann?

Hmmm... Ich werde es so erklaeren (und hoffe, dass du einen Führerschein besitzt, und keinen TESLA) 🙈  :

Es ist das gleiche Prinzip wie eines Relais: 
Wenn du die Scheinwerfer im Auto einschaltest, du betätigst einen Schalter auf der Konsole.
Dieser Schalter aktiviert ein Relais, dass die Scheinwerfer mit Strom versorgt.
Warum nutzt man ein Relais anstatt direkt den Schalter mit den Scheinwerfern zu verbinden?
Antwort: weil die Scheinwerfer verbrauchen 2 x 55W = 110W
Wuerden 110W durch den Schalter auf der Konsole gehen, dann muesste dieser Schalter zehn Mal so gross sein wie jetzt, die Strom Leitungen bis zum Schalter viel dicker, um die Stroeme von 110W auszuhalten!
Darum, Relais!

Die gleiche Funktion die Relais macht, macht der ADUM. 
Nur anstatt Strom, er macht es mit Daten.
ADUMs benutzt wenn man auf einer Seite ein System hat das auf eine Spannung von z.B. 12V laeuft, aber auf der andere Seite ein System mit anderer Spannung z.B. 3V funktioniert. Und du willst Daten Kommunikation zwischen diese zwei Systeme haben.
Wuerdest du diese zwei Systeme anschliessen wollen, wirst du das 3V System frittieren.
Es sit kein "Drama" auch ohne ADUM fuer uns, weil der Victron MPPT hat 5V, der ESP32 3V und der ESP32 kann die 5V "ertragen".

Das war die Erklaerung des ADUMs.

Jetzt komm der zusaetzliche Vorteil: 
Durch diese Trennung der Spannungen der zwei Systeme (Beispiel mit 12V und 3V) existiert kein Risiko das Stroeme weitergeleitet werden (galvanische Trennung).
Die “galvanische Trennung” ADUM1201 zwischen ESP32 und Victron zu haben ist wichtig weil falls eine Sicherung die Erde/Masse des PV-Systems trennt, dann sucht der Victron die “andere verfügbare Masse” und die wird die deines VE.Direct sein. 
Hier erklaert jemand warum (weil die Victrons die gleiche Masse nutzen): [Victron Masse](https://youtu.be/slAlCC5aIq4?t=310 )

**Reply by SW-Niko** [reply]
>Funktionsweise eines Adum 1201 wäre für mich interessant um zu verstehen können!

Am besten du googelst mal nach dem Datenblatt vom ADUM1201. Auf der ersten Seite gibt es einem Überblick.

>Wann wird sie benötigt:
Bei mehreren Verbindungen des Mikrocontrollers mit Gerätschaften wie:

Ich denke ... das beste ist, wenn ich was zusammenstelle. Ist vielleicht auch für andere nützlich. 
Am Wochenende habe ich etwas Zeit.

**Reply by SW-Niko** [reply]
Hallo @Snusme83,
ich denke diese Unterlagen sollen das generelle Problem darlegen.
Da die Anlagen so unterschiedlich sind, kann ich nur eine generelle Empfehlung geben.

[ParalleleMasse.pdf](https://github.com/user-attachments/files/17495879/ParalleleMasse.pdf)

**Reply by Snusme83** [reply]
Servus @SW-Niko ,
danke für deine Bemühungen die evtl. Problematik zu erklären.

In der PDF wird auch wieder so allgemein gesprochen. Klar, es gibt auch verschiedene Aufbauten des BKW (PV) in Verbindung mit Speicher (Pylontech, Autobatterien, DiY- Speicher,...)!

Im Grunde handelt es sich in diesem "Forum" aber immer um den ähnlichen Aufbau des grandiosen OpenDTUonBattery!

Ein in diesem PDF angesprochener Fehler kann eigentlich nur sein:

- Kappung der starken Stromleitungen
- Gaga- Verhalten eines BMS bei DIY- Speichers, weil (billige) BMS Masse/ GND trennen mit MOSFET bei Tiefentladung. Frage: Macht Pylontech das genauso?

So, jetzt haben wir noch die sog. Wartungsarbeiten:

Sollten diese erforderlich sein, trenne ich für mich:
- PV- Module (3 x Victron Smartsolar, zumindest bei Tageslicht),
- Pylontechs (ich mache die aus).

Gefahr ist dann eigentlich gebannt!🤷🏻‍♂️ Für mich wäre das an sich logisch und schlüssig.

Klar, wenn ich noch ein Adum1201 zwischen den Leitungen (VE.Direct Victron SmartSolar und ESP) habe, mache ich das Gesamtkonstrukt noch einmal ein Stück sicherer. Kann (werde) ich noch machen. Leider ist mir der Anschluss dieses Adum 1201 noch nicht in Gänze klar.

Nächstes ist die autarke (via Pylontech) Versorgung des Mikrocontrollers mittels Stepdown- Wandler. Da hilft ein Adum1201 nicht.
Lösung ist scheinbar ein DC-/ DC Wandler, welcher isoliert ist - habe aber bereits die hier empfohlenen bestellt (3 Stück).

Bin leider kein Elektriker (im nächsten Leben steht das zur Auswahl), deswegen die laienhaften Ausdrücke/ Erklärung.

**Reply by SW-Niko** [reply]
>Macht Pylontech das genauso?

Keine Ahnung. Ich habe keine Pylontech

>Leider ist mir der Anschluss dieses Adum 1201 noch nicht in Gänze klar.

Schon gesehen [Link](https://opendtu-onbattery.net/hardware/vedirect/)?

>Ein in diesem PDF angesprochener Fehler kann eigentlich nur sein:
    Kappung der starken Stromleitungen
    Gaga- Verhalten eines BMS bei DIY- Speichers, weil (billige) BMS Masse/ GND trennen mit MOSFET bei Tiefentladung. Frage: Macht Pylontech das genauso?

Hat der Victron SmartSolar eine Sicherung in der Masseleitung?

Und da gibt es noch viele weitere Sachen die man wissen sollte bevor man die galvanische Trennung weglässt.
Ich kenne meine verbauten Komponenten auch nicht bis ins Detail und darum versuche ich Risiko zu vermeiden und habe alle seriellen Schnittstellen galvanisch getrennt.

**Reply by Sorbat** [reply]
Hallo 👋
Besteht dieses Problem auch beim Lynx Distributor? 
Also wenn man Ihn mit externen 5V betreibt, dann sollte er doch nur an Plus hängen und damit eigentlich kein Problem darstellen, oder irre ich mich da?

**Reply by SW-Niko** [reply]
>Besteht dieses Problem auch beim Lynx Distributor

Jetzt musste ich erst mal Googeln was ein Lynx Distributor ist. 

![grafik](https://github.com/user-attachments/assets/dfe5082f-27ea-46d5-aaf0-e71eeb3b92f5)

Ok ... Er addiert zusätzliche Sicherungen in die Plus-Leitung.
Was würde der Lynx Distributor machen wenn die BMS die Masse-Leitung abschaltet? Fliegt dann zusätzlich eine Sicherung im Lynx Distributor? Wird dann verhindert das der Strom über andere Massen fließt?

Die Antworten solltest du mittlerweile selber kennen.

Du bekommst 3 Punkte wenn du auf "Nichts", "Nein" und "Nein" kommst.

> ..dann sollte er doch nur an Plus hängen

Also ich sehe auch - (Masse) Anschlüsse.

**Reply by Sorbat** [reply]
> Also ich sehe auch - (Masse) Anschlüsse.

Ja natürlich hat der Distributor auch  Masse Anschlüsse, es ist ja schließlich eine Busbar 😄

Aber ja, ich hatte einen Denkfehler. Die Platine, auf der die LEDs sitzen, ist natürlich auch an Masse angeschlossen. 

Dann baue ich die PCB lieber aus und hab halt keine bunten Lichter 😕

Ich finds nur etwas merkwürdig, warum Victron das so als fertiges Produkt verkauft, wenn das so brandgefährlich ist. Schließlich ist Victron ja auch keine kleine Hinterhofklitsche und die Stückzahlen liegen ja vermutlich auch eher in höheren Bereichen 🤷🏻‍♂️

**Reply by SW-Niko** [reply]
Ich mache mir langsam Sorgen..... 

> Ich finds nur etwas merkwürdig, warum Victron das so als fertiges Produkt verkauft, wenn das so brandgefährlich ist.

Ich sehe nichts Gefährliches an dem Lynx Distributor.
Das Problem bei parallelen Massen zwischen Geräten ist dir anscheinend noch nicht klar.
Ich will dir nicht auf die Füße treten, vielleicht ist es besser wenn du dir für die Elektrik professionelle Hilfe vor Ort suchst.

**Reply by hoschiking** [reply]
Victron verkauft nichts brandgefährliches, im Gegenteil, das Original Dantekabel von Victron (die USB-Variante) hat eine galvanische Trennung eingebaut, das erklärt unter anderem den Preis. Des weiteren kann Victron nichts dafür, wenn sich Leute DIY einen Speicher nach Youtube-Anleitung zusammenbauen und die Grundlagen der Elektrotechnik nicht eingehalten werden, da das notwedige Fachwissen fehlt. Victron verkauft ein Gerät, welches aber nicht Standalone betrieben wird, sondern am Ende nur ein Teil eines Gesamtsystems ist, da ist der Hersteller des Gesamtsystems in der Pflicht.

**Reply by Sorbat** [reply]
> Ich will dir nicht auf die Füße treten, vielleicht ist es besser wenn du dir für die Elektrik professionelle Hilfe vor Ort suchst.

Sorry, ganz übersehen. Würde ich sogar erwägen, wenn ich nicht so unzufrieden mit der Arbeit des Elektrikers gewesen wäre und die Hälte nochmal neu machen musste. 

> das Original Dantekabel von Victron (die USB-Variante) hat eine galvanische Trennung

meine Frage bezog sich auf den Distributor, nicht auf das Datenkabel. 

Es ist mir aber auch zu kräftezehrend mir Gedanken über den Distributor zu machen, dessen Funktion eigentlich überflüssig ist.

**Reply by DonJohnLong** [reply]
Meine Victrons sind alle mit einem Adum1201 galvanisch getrennt, das DC/DC Netzteil war der Schelm der zum Kabelbrand geführt hat.

EDIT: Adums sind leider für höhere Spannungen nicht geeignet.

**Reply by SW-Niko** [reply]
Hallo @DonJohnLong,
das interessiert mich jetzt.
Alle Victron galvanisch getrennt. DCDC nicht galvanisch getrennt. 
Und die Verbindung BMS zu ESP32 ebenfalls nicht galvanisch getrennt?

**Reply by hoschiking** [reply]
Gerade die vom BMS zum ESP muss galvanisch getrennt sein, zumindest wenn man die UART verwendet. Bei RS485 sieht es anders aus, da ist die Trennung schon im Übertrager implementiert.

**Comment by Manos1966** [comment]
Hi @hoschiking 
>das Original Dantekabel von Victron (die USB-Variante) hat eine galvanische Trennung eingebaut, das erklärt unter anderem den Preis. 

das wuerde den Preis erklaeren 👍  Ich weiss aber nicht, wie Victron eine galvanische Trennung im Kabel haette bauen koennen.
Hast du vielleicht Infos darueber?

**Reply by hoschiking** [reply]
Ja, das Datenblatt des Kabels, da steckt es drin. Ich meinte ja die USB Variante, da ist ja sowieso ein USB -UART Wandler eingebaut, das ist der dicke Knubbel in der Mitte des Kabels. Ich denke, da wird auch die galvanische Trennung eingebaut sein

**Reply by Manos1966** [reply]
Notiz zu selbst: Ich muss aufhoeren auf dem Handy Git Postings zu lesen.... 🤦‍♂️ 

Ja klar, USB 😄 
Aber mit diesem Kabel kann ich nicht Victron MPPT und OpenDTU verbinden 😞

**Reply by hoschiking** [reply]
Bestell dir doch den ADUM, damit ist die galvanische Trennung doch sehr einfach gebaut.

**Reply by SW-Niko** [reply]
>Notiz zu selbst: Ich muss aufhoeren auf dem Handy Git Postings zu lesen.... 🤦‍♂️
🤣 

>..dicke Knubbel in der Mitte des Kabels. 

Ja genau. Und wen mehrere MPPTs an ein Victron Steuergerät angeschlossen werden dann ist die galvanische Trennung im Steuergerät zu finden.

![grafik](https://github.com/user-attachments/assets/c163c0c7-6760-4aec-aef7-ade2cb54e327)
 
Ich baue mir die Teile selber zusammen.

![grafik](https://github.com/user-attachments/assets/621b0d08-de60-414a-9753-43227ee8d1eb)

Aber dafür muss man eben etwas löten können.
Ich glaube ich habe schon mal nach einem UART <> UART Kabel mit galvanischer Trennung gesucht, das man einfach fertig kaufen kann. Leider habe ich nichts passendes gefunden.
